internal/controller/validator_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"testing"
 
 	"github.com/go-logr/logr"
-	"github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
```

Comment:
This change caused small issue in CI:
```
internal/controller/validator_test.go:7:2: redundant-import-alias: Import alias "v1" is redundant (revive)
	v1 "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
```

Reply:
golang forces an alias if I try removing it, so set it to velerov1

Reply:
what you mean by "golang forces an alias"?

Reply:
Removing the explicit v1 package import alias didn't work because it kept adding it back when the file was saved. Not sure what does that

Reply:
that is not Go, probably your IDE

in VSCode, for example, this setting is required to avoid this
```json
"[go]": {
        "editor.codeActionsOnSave": {
            "source.organizeImports": "never"
        }
    },
```